### ISSUE TITLE
Move DISSINET main deploy from MUNI Arts server to e-infra cloud

### ISSUE THREAD

**Issue by davidzbiral** [DISSINET/InkVisitor#2160]
- Move DISSINET main deploy from MUNI Arts server to e-infra cloud.
- Run it from the same [DISSINET page](https://dissinet.cz/apps/inkvisitor/) (or, if impossible or highly unusual, create a redirect).

**Comment by davidzbiral** [comment]
@jancimertel @adammertel Very intense work expected for 27-31 May, and last time it slowed us down by easily 50% of the time. Realistic to do the move quickly or not and needs a lot of testing or something?

**Comment by adammertel** [comment]
@davidzbiral we do not think this is a good idea as:
 - we do not expect heavy slow-downs this time as we have been fixing possible causes for that
 - the pipeline for the deployment on the cluster is not ready yet
 - the data backup for the e-infra is not integrated and tested yet
 - its not that simple to merge changes from two sources together - so when we decide to move on e-infra, the environment should be tested as there is no simple way back

**Comment by davidzbiral** [comment]
@adammertel OK, let's give it more time, anyway the period of intense work (= this week) is not good for this move.

**Comment by adammertel** [comment]
@davidzbiral we have been postponing the cluster deployment as we thought it's not a priority now. Should we prioritize it?

**Comment by davidzbiral** [comment]
@adammertel Depends on Annotator and Explorer core functionalities. But @tomaham tells me that for Niort data publication cloud is more appropriate.

**Comment by adammertel** [comment]
Okey so the idea would be to deploy the Niort instance on the cloud as the first step?

**Comment by davidzbiral** [comment]
@adammertel Yes, this is the more immediate need for developping this than moving the main deploy, currently - tomaham thinks this should not be on our server.

**Comment by davidzbiral** [comment]
Done today, thanks!
